news/2024/2024-05-01-the-nat-gazette-june-2024.md

```diff
@@ -0,0 +1,124 @@
+
+#### A fresh application workflow
+
+The application process has been streamlined, easing users into the application process by giving more specific pointers on what mods and explanations they're expected to give at the time of applying. The goal of this change was to "handhold" new applicants through the application process and for the NAT to focus more on the decision-making of candidates rather than solely base their verdict from mods alone.
+
+#### Removal of the ranking criteria test
+
+Yep, you heard it here first. Or probably not. But the RC test is gone, dusted. Vanished, not to come again in the foreseeable future. The test in its current form didn't offer a lot. Besides being a filter and an English comprehension test, it was redundant as applicants would better show their understanding of the Ranking Criteria through their mods. 
+
+#### Streamlining cooldowns
+
+Cooldowns have been homogenized across the board. Failed applicants and BNs who get kicked for low activity will always have to wait 60 days before reapplying again under normal circumstances. This does not apply to Beatmap Nominators who leave on good terms and want to rejoin within a year after their departure who will be able to rejoin instantly.
+
+#### Progress bar for applications
+
+One of the issues with the opaqueness of the process was that candidates didn't know when their verdict would be delivered, or at what stage their application was. By offering our very own BN app parcel tracking service, applicants will now be able to be updated in real time when their food — I mean, their application — be delivered.
+
+Insert image perchance.
+
+#### Reworking application feedback
+
+Feedback has been simplified. In short, this change will make the feedback writing process which was deemed redundant be less time-consuming. This will quicken the time for the results of an application to be delivered. The evaluators' comments will instead serve as more in depth feedback.
+
+#### Communication platform post-applications (need a better title) (I couldn't come up with one either)
+
+After receiving their verdict, users can reply to their evaluators' feedback in the BN site. While the goal of this change is not to facilitate appeals as under normal circumstances the result of an application is unchangeable, applicants are encouraged to ask questions and clarification on their feedback by using this new feature.
+
+### Within the Beatmap Nominators
+
+#### Probation is no longer a demotion option
+
+The Probation Beatmap Nominators user group has been reworked slightly. It will now serve its original goal, which is as a trial ground for green beards! When a new Beatmap Nominator is allowed into the team, they will be put in probation for their first couple months. After evaluating their performance with actual BN work, they will either be promoted or removed. Full Beatmap Nominators cannot under any circumstance be demoted back into probation.
```

Comment:
```suggestion
The Probationary Beatmap Nominators user group has been reworked slightly. It will now serve its original goal, which is as a trial ground for greenhorns! When a new Beatmap Nominator is allowed into the team, they will be put in probation for their first couple months. After evaluating their performance with actual BN work, they will either be promoted or removed. Full Beatmap Nominators cannot under any circumstance be demoted back into probation.
```